app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java

```diff
@@ -24,65 +27,81 @@
  * @author juan.herrera
  * @since 31/08/2016
  */
-public class SubscriberDetailFragment extends Fragment implements SubscriberDetailView {
+public class SubscriberDetailFragment extends Fragment implements SubscriberDetailView, RepositoryAdapter.ItemRepoClickListener, View.OnClickListener {
 
-    private ImageView imgAvatar;
-    private TextView txtFullName;
-    private TextView txtUserName;
-    private TextView txtCompany;
-    private TextView txtLocation;
-    private TextView txtFollowingCounter;
-    private TextView txtReposCounter;
+    private static final String ARG_USERNAME = "ARG_USERNAME";
+    private String userNameArg;
```

Comment:
I think this variable is a litle confusing with the previous static one. Rename it to just `userName`